source/_posts/release-1.0.0-beta.3.md

```diff
@@ -3,7 +3,7 @@ title: "Release: 1.0.0-beta.3"
 date: 2019-06-05
 ---
 
-Front-Commerce `1.0.0-beta.3` has been released with several improvements: bla bla bla…
+Front-Commerce `1.0.0-beta.3` has been released with several improvements: better embedded payment experience, customizable WYSIWYG components, configurations override…
```

Comment:
```suggestion
Front-Commerce `1.0.0-beta.3` has been released with several improvements: better embedded payment experience, customizable WYSIWYG components, configurations override…

We also want to thank our partner [PH2M](https://www.ph2m.com) for their first PRs, and [Webqam](https://www.webqam.fr) and [Occitech](https://www.occitech.fr) for their contributions. This release is the first one with PR merged from partners!
```

Reply:
To be fully honest, there was one for a typo (by PH2M) beforehand.